packages/terra-time-input/src/TimeInput.scss

```diff
@@ -1,40 +1,204 @@
-      display: inline-block;
-      font-size: 1em;
-      margin: 0;
-      padding: 0.1em 0;
+  @mixin focus-styles {
+    border-color: var(--terra-time-input-focus-keyboard-border-color, #4cb2e9);
+    border-style: var(--terra-time-input-focus-keyboard-border-style, solid);
+    border-width: var(--terra-time-input-focus-keyboard-border-width, 1px);
+    opacity: var(--terra-time-input-focus-keyboard-opacity, 1);
+    outline: none;
+  }
+
+  .time-input {
+    background-color: var(--terra-time-input-background, #fff);
+    background-image: var(--terra-time-input-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-time-input-focus-background-start, 0 0);
+    border-color: var(--terra-time-input-desktop-focus-border-color, #dedfe0);
+    border-radius: var(--terra-time-input-border-radius, 3px);
+    border-style: var(--terra-time-input-desktop-focus-border-style, solid);
+    border-width: var(--terra-time-input-desktop-focus-border-width, 1px);
+    direction: ltr;
+    display: inline-block;
+    font-size: var(--terra-time-input-font-size, 1em);
+    margin: 0;
+    padding-bottom: var(--terra-time-input-padding-bottom, 0.286em);
+    padding-left: var(--terra-time-input-padding-left, 0.214em);
+    padding-right: var(--terra-time-input-padding-right, 0.214em);
+    padding-top: var(--terra-time-input-padding-top, 0.286em);
+    transition-duration: var(--terra-time-input-focus-transition-duration, 0.2s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-time-input-focus-transition-timing-function, ease);
+
+    &.is-focused {
+      @include focus-styles;
+      background-size: var(--terra-time-input-keyboard-animation-size, 0 0);
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-time-input-focus-keyboard-box-shadow-ltr, 0 0 8px #a6d9f4);
+      }
 
-      &.is-focused {
-        border-color: #26a2e5;
-        outline: none;
+      [dir=rtl] & {
+        box-shadow: var(--terra-time-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
       }
     }
+  }
+
+  .time-input-group {
+    margin-left: var(--terra-time-input-time-input-group-margin-left, 0.357em);
+    margin-right: var(--terra-time-input-time-input-group-margin-right, 0.357em);
+
+    &:first-of-type {
+      margin-left: 0;
+    }
 
-    .time-input-hour,
-    .time-input-minute {
-      border: none;
+    &:last-of-type {
+      margin-right: 0;
+    }
+  }
+
+  .time-input-hour,
+  .time-input-minute {
+    background-color: var(--terra-time-input-background, #fff);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-time-input-focus-background-start, 0 0);
+    border-color: var(--terra-time-input-desktop-focus-border-color, #dedfe0);
+    border-radius: var(--terra-time-input-border-radius, 3px);
+    border-style: var(--terra-time-input-desktop-focus-border-style, solid);
```

Comment:
Same comment as the one about `-focus-` named variables in `.time-input` applies to the `.time-input-hour` and `.time-input-minute` selectors.

Reply:
Done. https://github.com/cerner/terra-core/pull/1027/commits/1d59ce24cd77ec2a62eaeb02e086047c01bdf25a

Reply:
Added the themeable-variables right here. https://github.com/cerner/terra-core/pull/1027/commits/16e498e27d904c67a851918dbfcae1683bb75c64